generators/chipyard/src/main/resources/csrc/cospike.cc

```diff
@@ -104,6 +111,27 @@ extern "C" void cospike_cosim(long long int cycle,
     uint64_t default_boot_addr = 0x80000000;
     boot_addr_reg->store(0, 8, (const uint8_t*)(&default_boot_addr));
 
+    for (auto& mem : mems) {
+      if (mem.first == info->mem0_base) {
+        std::string path_name = "chipyard-cosim-" + std::to_string(getpid());
```

Comment:
Remove the memory matching stuff, this needs the SimDRAM memory channel dupllication issue to be fixed.